pkg/pillar/types/clustertypes.go

```diff
@@ -41,6 +41,7 @@ type ENClusterAppStatus struct {
 	IsDNSet             bool      // DesignatedNodeID is set for this node
 	ScheduledOnThisNode bool      // App is running on this device
 	StatusRunning       bool      // Status of the app in "Running" state
+	StatusStopped       bool      // App is in stopped state
```

Comment:
From a quick read-through I think it looks ok but why do we need both a StatusRunning and StatusStopped bool?

Reply:
I modified this commit a bit and retesting. Will amend the commit ASAP

Reply:
Closing this because I created PR https://github.com/naiming-zededa/eve/pull/392